### PR TITLE
Pullin

### PR DESCRIPTION
**Pull request by Tjudge1**
etc

### CODE REVIEW

**Review by Tjudge1 — COMMENTED** [review]
makeown description adjustment

File: changelog.txt [hunk 1]
```diff
@@ -26,6 +26,7 @@ that repo.
 - `gui/gm-editor`: they key column now auto-fits to the widest key
 - `prioritize`: revise and simplify the default list of prioritized jobs -- be sure to tell us if your forts are running noticeably better (or worse!)
 - `gui/control-panel`: add `faststart` to the system services
+- `makeown`: minor adjustment made to documentation, description, and usage.
```
Comment (myk002) [inline changelog.txt:29]: this is minor enough that it doesn't really need a changelog entry.

File: docs/makeown.rst [hunk 2]
```diff
@@ -2,11 +2,11 @@ makeown
 =======
 
 .. dfhack-tool::
-    :summary: Converts the selected unit to be a member of your fortress.
+    :summary: Converts the selected unit to be a member of your fortress. Adds them as a citizen.
```
Comment (myk002) [inline docs/makeown.rst:5]: a summary can only have one sentence. the description is parsed up to the first period. How about "Converts the selected unit to be a citizen of your fortress"?

**Comment by myk002** [comment]
Thanks!

**Review by myk002 — APPROVED** [review]
Thanks!

**Pull request merged**